contao/assets/scss/style.scss

```diff
@@ -0,0 +1,161 @@
+/**
+ * This file is part of MetaModels/core.
+ *
+ * (c) 2012-2019 The MetaModels team.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * This project is provided in good faith and hope to be usable by anyone.
+ *
+ * @package    MetaModels
+ * @subpackage Core
+ * @author     Andreas Isaak <info@andreas-isaak.de>
+ * @author     Ingolf Steinhardt <info@e-spin.de>
+ * @author     Sven Baumann <baumann.sv@gmail.com>
+ * @author     Carolina Koehn <ck@kikmedia.de>
+ * @copyright  2012-2019 The MetaModels team.
+ * @license    https://github.com/MetaModels/core/blob/master/LICENSE LGPL-3.0-or-later
+ * @filesource
+ */
+
+//urls
+$icon_fields: url('../images/icons/fields.png');
+$icon_dca_add: url('../images/icons/dca_add.png');
+$icon_rendersettings_add: url('../images/icons/rendersettings_add.png');
+$icon_pal_open: url('../../../../themes/default/images/palOpen.gif');
+$icon_filter_setting: url('../images/icons/filter_settings.png');
+$icon_locale: url('../images/icons/locale.png');
+
+body[class*="version_3"] {
+	a[class*="header_"] {
+		margin-left: 15px;
+	}
+	.tl_file_list .field_type img {
+		margin-right: 8px;
+		float: left;
+	}
+}
+.header_css_fields {
+	padding: 2px 0 3px 20px;
+	background-image: $icon_fields;
+	background-position: left center;
+	background-repeat: no-repeat;
+}
+.header_add_all {
+	padding: 2px 0 3px 20px;
+	background-image: $icon_dca_add;
+	background-position: left center;
+	background-repeat: no-repeat;
+}
+.rendersetting_add_all {
+	background-image: $icon_rendersettings_add;
+}
+.dca_palette {
+	color: rgba(138, 184, 88, 1);
+	margin: 6px 0;
+	padding-left: 24px;
+	background: $icon_pal_open 3px center no-repeat;
+}
+.mm_problem_display {
+	margin-bottom: 30px;
+	ul {
+		padding: 0;
+		list-style: none;
+	}
+}
+.tl_subdca {
+	> legend {
+		margin: 0;
+		padding: 10px 0 10px 23px;
+		background: $icon_filter_setting no-repeat left center;
+	}
+	legend label {
+		font-weight: bold;
+	}
+}
+body[class*="version_2"] {
```

Comment:
I wonder if this is really still needed as we are Version 3/4 now.

Reply:
I think we can delete all `body[class*="version_N"] {`

Reply:
You are absolutely right. I will update the PR tomorrow.

Reply:
?

Reply:
@kikmedia which "tomorrow"?

Reply:
@kikmedia ping?

Reply:
we wait a long time :(

Reply:
fixed in https://github.com/MetaModels/core/pull/1286